targets/ChibiOS/_nanoCLR/nanoFramework.Hardware.Pico/nanoFramework_hardware_pico_native_PioIrqDriver.cpp

```
@@ -0,0 +1,164 @@
+        // write-1-to-clear
+        pio->IRQ = flags;
+    }
+
+    chSysUnlockFromISR();
+
+    NATIVE_INTERRUPT_END
+}
+
+// ---- NativeEventDispatcher driver procs -------------------------------------
+
+static HRESULT PioIrqInitialize(CLR_RT_HeapBlock_NativeEventDispatcher *pContext, unsigned __int64 userData)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)